modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_coordinates.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
     - field.storage.node.field_location_coordinates
     - node.type.branch
   module:
+    - datalayer
```

Comment:
can we include https://www.drupal.org/project/datalayer/issues/3102358 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

Reply:
Actually datalayer added here because it enabled to expose in third_party_settings rows below https://github.com/ymcatwincities/openy/pull/1931/files/17e950a3f51101947824ab8c34eedc767430818a#diff-b3d808e3a07850f153acb9cbc0c49814R12